### ISSUE TITLE
What is the difference between kitti and eigen？

### ISSUE THREAD

**Issue by xielinjiang** [mrharicot/monodepth#148]
@mrharicot
hi, thanks for your share. there are still some questions i cant understand.
1.What is the difference between kitti and eigen？

**Comment by andraghetti** [comment]
They are different test datasets. 
kitti means kitti2015 test (kitti_stereo_2015_test_files.txt) which contains 200 stereo images.
eigen means eigen test split (eigen_test_files.txt) which contains 697 images "sampled" from kitti raw data.

**Comment by mrharicot** [comment]
The training dataset splits are also different, to make sure no training image are in the same sequences as the test set.
Please see section 4.2 in the paper:
https://arxiv.org/pdf/1609.03677.pdf
